tests/test_sft_trainer.py

```diff
@@ -190,31 +174,30 @@ def test_resume_training_from_checkpoint_with_flag_false():
         assert init_trainer_state is not None
 
         # Get Training log entry for epoch 1
-        init_training_logs = _get_training_logs(tempdir, epoch=1)
+        init_training_logs = _get_training_logs_by_epoch(tempdir, epoch=1)
         assert len(init_training_logs) == 1
 
-        # Resume training with higher epoch and same output dir
+        # Training again with higher epoch and same output dir
         train_args.num_train_epochs += 5
         sft_trainer.train(MODEL_ARGS, DATA_ARGS, train_args, None)
         _validate_training(tempdir)
 
         # Get Training log entry for epoch 1
-        final_training_logs = _get_training_logs(tempdir, epoch=1)
+        final_training_logs = _get_training_logs_by_epoch(tempdir, epoch=1)
         assert len(final_training_logs) == 2
 
 
 def test_resume_training_from_checkpoint_with_flag_checkpoint_path():
```

Comment:
SHould include LoRA here
```suggestion
def test_resume_training_from_checkpoint_with_flag_checkpoint_path_lora():
```